src/Compilers/CSharp/Portable/Syntax/TypeDeclarationSyntax.cs

```diff
@@ -18,6 +18,18 @@ public int Arity
                 return this.TypeParameterList == null ? 0 : this.TypeParameterList.Parameters.Count;
             }
         }
+
+        public new TypeDeclarationSyntax AddAttributeLists(params AttributeListSyntax[] items)
```

Comment:
>AddAttributeLists [](start = 41, length = 17)

Did not find any reference to this.
Same for `AddModifiers`.
Seems like an existing test gap on public API. Could you add a test?

Reply:
test added.